stdlib/src/builtin/object.mojo

```diff
@@ -572,29 +572,25 @@ struct _ObjectImpl(
             writer.write("None")
             return
         if self.is_bool():
-            writer.write(str(self.get_as_bool()))
+            writer.write(self.get_as_bool())
             return
         if self.is_int():
-            writer.write(str(self.get_as_int()))
+            writer.write(self.get_as_int())
             return
         if self.is_float():
-            writer.write(str(self.get_as_float()))
+            writer.write(self.get_as_float())
             return
         if self.is_str():
             writer.write(
-                "'"
-                + str(
-                    StringRef(
-                        self.get_as_string().data, self.get_as_string().length
-                    )
-                )
-                + "'"
+                "'",
+                StringRef(
+                    self.get_as_string().data, self.get_as_string().length
+                ),
```

Comment:
```suggestion
                self.get_as_string(),
```

Reply:
I was also thinking about that, should of tried it. I wonder why there was so much indirection before...

Reply:
Tests failed, maybe it was the formatting, hang on

Reply:
It's really that `get_as_string` doesn't return a `String` but `_ImmutableString`.

Reply:
Oh yeah, it's because it return an `_ImmutableString`

Reply:
So the original should be fine and not do an extra allocation.